src/Core/src/HotReload/HotReloadExtensions.cs

```diff
@@ -36,6 +37,7 @@ public static void CheckHandlers(this IView? view)
 
 		public static List<MethodInfo> GetOnHotReloadMethods(this Type type) => getOnHotReloadMethods(type).Distinct(new ReflectionMethodComparer()).ToList();
 
+		[UnconditionalSuppressMessage("Trimming", "IL2080", Justification = TrimmerHelper.HotReload)]
```

Comment:
I basically ignored trimmer warnings on anything "hot reload", should be fine?

Reply:
Yep. Not supported in a release app.